pom.xml

```diff
@@ -24,75 +24,150 @@
-  </pluginRepositories>
-
-  <distributionManagement>
-    <site>
-      <id>github-project-site</id>
-      <url>gitsite:git@github.com/jenkinsci/hello-world-plugin</url>
-    </site>
-  </distributionManagement>
-
-  <properties>
-    <jenkins.version>1.580.1</jenkins.version>
-    <java.level>6</java.level>
-  </properties>
+    <artifactId>hello-world</artifactId>
+    <version>2.1-SNAPSHOT</version>
+    <packaging>hpi</packaging>
+
+    <name>Hello World Example Plugin</name>
+    <description>Jenkins World 2017 Intro Plugin</description>
+    <url>http://wiki.jenkins-ci.org/display/JENKINS/Hello+World+Plugin</url>
+    <!-- The default licence for Jenkins OSS Plugins is MIT. Substitute for the applicable one if needed. -->
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>http://opensource.org/licenses/MIT</url>
+        </license>
+    </licenses>
+
+    <!-- If you want this to appear on the wiki page:
+    <developers>
+        <developer>
+            <id>kohsuke</id>
+            <name>Kohsuke Kawaguchi</name>
+            <email>kk@kohsuke.org</email>
+        </developer>
+        <developer>
+            <id>jieryn</id>
+            <name>Jesse Farinacci</name>
+            <email>jieryn@gmail.com</email>
+        </developer>
+    </developers>
+    -->
+
+    <!-- Assuming you want to host on @jenkinsci:
+    <scm>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+    </scm>
+    -->
+
+    <properties>
+        <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
+        <jenkins.version>2.60.3</jenkins.version>
+        <java.level>8</java.level>
+        <!-- Other properties you may want to use:
+             ~ jenkins-test-harness.version: Jenkins Test Harness version you use to test the plugin. For Jenkins version >= 1.580.1 use JTH 2.0 or higher.
+             ~ hpi-plugin.version: The HPI Maven Plugin version used by the plugin..
+             ~ stapler-plugin.version: The Stapler Maven plugin version required by the plugin.
+        -->
+    </properties>
+
+    <repositories>
+        <repository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </repository>
+    </repositories>
+
+    <pluginRepositories>
+        <pluginRepository>
+            <id>repo.jenkins-ci.org</id>
+            <url>https://repo.jenkins-ci.org/public/</url>
+        </pluginRepository>
+    </pluginRepositories>
+
+    <distributionManagement>
+        <site>
+            <id>github-project-site</id>
+            <url>gitsite:git@github.com/jenkinsci/hello-world-plugin</url>
+        </site>
+    </distributionManagement>
+
+    <build>
```

Comment:
109-121 can / should be removed as you have set the `java.level` to 8 above which is the way to do it with the parent pom

Reply:
Nice suggestion!  I'll remove that block.  One more thing to use from the parent pom, one less thing to maintain in the plugin pom.